src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php

```diff
@@ -48,8 +48,8 @@ public function __construct($savePath = null)
             $baseDir = ltrim(strrchr($savePath, ';'), ';');
         }
 
-        if ($baseDir && !is_dir($baseDir)) {
-            mkdir($baseDir, 0777, true);
+        if ($baseDir && !is_dir($baseDir) && !@mkdir($baseDir, 0777, true) && !is_dir($baseDir)) {
+            throw new \RuntimeException(sprintf('The Session Storage was not able to create a directory: %s', $baseDir));
```

Comment:
Ok then , thanks for the argument :)
I suggest rewording to remove the `:` in the message: 
`The Session Storage was not able to create directory "%s".`